### PR TITLE
DCO and CoC (updated)

### PR DESCRIPTION
**Pull request by refack**
Updated https://github.com/nodejs/node-gyp/pull/771
lifted verbatim from
https://github.com/nodejs/node/blob/master/CONTRIBUTING.md
then `/s/Node.js/node-gyp/`

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# Contributing to node-gyp
+
+## Code of Conduct
+
+Please read the
+[Code of Conduct](https://github.com/nodejs/TSC/blob/master/CODE_OF_CONDUCT.md)
+which explains the minimum behavior expectations for Node.js contributors.
```
Comment (addaleax) [inline CONTRIBUTING.md]: maybe s/Node.js/node-gyp/?

**Comment by gibfahn** [comment]
Where did this text come from? Might be worth including it in the commit message.

File: CONTRIBUTING.md (hunk 1 above)
Comment (refack) [inline CONTRIBUTING.md]: ack

**Comment by refack** [comment]
> Where did this text come from? Might be worth including it in the commit message.

Ack

**Comment by gibfahn** [comment]
Landed in https://github.com/nodejs/node-gyp/commit/2e40fad2bf50dc41acdeb10edaf3b7d64b411d0b

**Pull request closed without merging**